packages/labs/src/components/select/select.tsx

```diff
@@ -159,7 +178,7 @@ export class Select<T> extends React.Component<ISelectProps<T>, ISelectState<T>>
         // not using defaultProps cuz they're hard to type with generics (can't use <T> on static members)
         const { filterable = true, disabled = false, inputProps = {}, popoverProps = {} } = this.props;
 
-        const { ref, ...htmlInputProps } = inputProps;
+        const { ref, value, ...htmlInputProps } = inputProps;
```

Comment:
move the `value` prop below the spread (swap lines 188 and 189) and you won't have to do this.

how to JSX spread:

```jsx
<Tag
    overridableProps...
    {...spread}
    notOverridableProps...
/>
```